### DISCUSSION TITLE
A more interactive way to upvote and comment on posts

### DISCUSSION THREAD

**Discussion by AnishDe12020** [dailydotdev/daily#492] [Feature Requests]
Right now, we click on an article, we are taken to that article's page, we read the article but most don't give the effort to go back to their daily.dev feed, find the article, and then upvote it or add a comment.

Now, through the browser extension, daily.dev can put an overlay on the site of the article we are reading from where we can directly upvote it and comment on it. 

Potential solution - when we are getting redirected from api.daily.dev, it lets the browser extension know that we have opened that article. The browser extension will find that tab and maybe put a small button that we can press to activate the aforementioned overlay. Maybe also once one is done reading the article (scroll to bottom), it can automatically activate the overlay.

**Comment by idoshamun** [comment]
Hi @AnishDe12020,

All I have to say is stay tuned 👀

**Reply by AnishDe12020** [reply]
Sure!
